tsconfig.json

```diff
@@ -1,19 +1,11 @@
 {
+    "extends": "tsconfigs/browser-module",
     "compilerOptions": {
+        "esModuleInterop": false,
+        "strictFunctionTypes": false,
         "target": "ES5",
-        "noImplicitAny": true,
-        "sourceMap": true,
-        "strictNullChecks": true,
-        "declaration": true,
-        "removeComments": false,
-        "noUnusedLocals": true,
```

Comment:
This is more suitable for linting. The TypeScript team feel this way. I won't go digging the comments now. We have linting now.